t/full-stack.t

```diff
@@ -399,6 +406,7 @@ subtest 'Cache tests' => sub {
 
     like($log_content, qr/Failed to download.*non-existent.qcow2/, 'Test 8 failure message found in log');
     like($log_content, qr/Result: setup failure/,                  'Test 8 state correct: setup failure');
+
```

Comment:
that I see as an unrelated change

Reply:
When removing this blank line, can I remove the blank line between the `like` calls? It makes most sense to either split this block or not split it at all. And it is not that unrelated considering I'm improving the fullstack test here. I can make it a separate commit, though.